config/database.yml

```diff
@@ -23,6 +23,7 @@ test:
   username: <%= ENV['DATABASE_USERNAME'].presence || "root" %>
   password:  <%= ENV['DATABASE_PASSWORD'].presence || "" %>
   socket: <%= ENV['DATABASE_SOCKET'] || ["/var/run/mysqld/mysqld.sock", "/opt/local/var/run/mysql5/mysqld.sock", "/tmp/mysql.sock"].find { |path| File.exist? path } %>
+  host: <%= ENV['DATABASE_HOST'] || "" %>
```

Comment:
Allows tests to be run against non local database (see also: https://github.com/huginn/huginn/issues/1979)